src/main/java/org/reactome/addlinks/AddLinks.java

```diff
@@ -416,6 +412,8 @@ private void purgeUnusedRefDBs()
 	@SuppressWarnings("unchecked")
 	private void createReferences(long personID, Map<String, Map<String, ?>> dbMappings) throws IOException, Exception
 	{
+		String linkCheckReportName = LINK_CHECK_REPORTS_PATH + "/linkCheckSummaryReport" + DateTimeFormatter.ofPattern(DATE_PATTERN_FOR_FILENAMES).format(LocalDateTime.now()) + ".tsv";
+		Files.write(Paths.get(linkCheckReportName), "RefDBName\tNumOK\tNumNotOK\n".getBytes(), StandardOpenOption.CREATE);
```

Comment:
Should a StandardOpenOption be added to handle the case if the file already exists (i.e. overwrite or append to the end of the file)?

Reply:
The file name should have the datetime, specific down to the second, it is unlikely there will be another file already there with the same name.